include/io/mman-win32/mman.h

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
That's right, I forgot about Google code going away... I guess we will just leave it.